test/functional/toolchains/cmake/cmakedeps2/test_cmakeconfigdeps_new.py

```diff
@@ -707,6 +725,102 @@ def package_info(self):
         assert "module: Release!" in c.out
         assert "vector: Release!" in c.out
 
+    @pytest.mark.skip()
+    def test_libs_components_multilib_component(self):
+        """
+        cpp_info.components["mycomp"].libs = ["lib1", "lib2"]
+        """
+        c = TestClient()
+
+        from conan.test.assets.sources import gen_function_h
+        vector_h = gen_function_h(name="vector")
+        from conan.test.assets.sources import gen_function_cpp
```

Comment:
They're already imported:

```suggestion
        vector_h = gen_function_h(name="vector")
```

Reply:
Done